cuda/extensions.bzl

```diff
@@ -148,15 +220,37 @@ def _impl(module_ctx):
 
     for _, toolkit in registrations.items():
         if components_mapping != None:
-            cuda_toolkit(name = toolkit.name, components_mapping = components_mapping, version = redist_version)
+            # Always use the maximum version so the toolkit includes all components.
```

Comment:
This is not quite true if CTK delete some component in the future. I think a union across all CTK versions will be a little bit more robust.

Reply:
Lets leave it for future improvement, just point it out :)